sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/implementation/helpers/Utility.java

```diff
@@ -111,101 +35,13 @@ public static GeoPosition fromCommaSeparatedString(String position) {
         return null;
     }
 
-    /**
-     * Returns a string representation of a {@link GeoPosition}.
-     *
-     * @return string representation
-     */
-    public static String positionToString(GeoPosition position) {
-        return position.getLatitude() + "," + position.getLongitude();
-    }
-
-    /**
-     * Converts a {@link BoundingBoxCompassNotation} into a {@link GeoBoundingBox}
-     *
-     * @param internalBoundingBox
-     * @return a GeoBoundingBox.
-     */
-    public static GeoBoundingBox toGeoBoundingBox(BoundingBoxCompassNotation internalBoundingBox) {
-        Objects.requireNonNull(internalBoundingBox, "Internal bounding box model is needed.");
-        Objects.requireNonNull(internalBoundingBox.getNorthEast(), "Top right of the bounding box is needed.");
-        Objects.requireNonNull(internalBoundingBox.getSouthWest(), "Bottom left of the bounding box is needed.");
-
-        final String northEastString = internalBoundingBox.getNorthEast();
-        final String southWestString = internalBoundingBox.getSouthWest();
-        final GeoPosition topRight = stringToCoordinate(northEastString);
-        final GeoPosition bottomLeft = stringToCoordinate(southWestString);
-
-        return new GeoBoundingBox(bottomLeft.getLongitude(), bottomLeft.getLatitude(),
-            topRight.getLongitude(), topRight.getLatitude());
-    }
-
-    /**
-     * Converts a comma-separated string into a {@link GeoPosition}.
-     *
-     * @param s - string to convert
-     * @return a GeoPosition
-     */
-    private static GeoPosition stringToCoordinate(String s) {
-        final String[] coordinateString = s.split(",");
-
-        if (coordinateString.length == 2) {
-            return new GeoPosition(Double.parseDouble(coordinateString[1]),
-                Double.parseDouble(coordinateString[0]));
+    public static GeoPosition fromDoubleList(List<Double> position) {
+        if (position.size() == 2) {
+            return new GeoPosition(position.get(0), position.get(1));
         }
-
         return null;
```

Comment:
Just returning `null` here may not be the best as this could be a larger problem in the response data.